### PR TITLE
Cleanup db function names

### PR DESCRIPTION
**Pull request by freeekanayaka**
This is a first batch of renames of db functions, to make them more consistent with current naming conventions in LXD.

Other similar renames will be done progressively in follow-up branches.

### CODE REVIEW

File: lxd/db/containers.go [hunk 1]
```diff
@@ -169,8 +169,8 @@ type InstanceBackupArgs struct {
 	CompressionAlgorithm string
 }
```
Comment (tomponline) [inline lxd/db/containers.go:170]: @freeekanayaka can this file be renamed to instances.go?

Comment (freeekanayaka) [reply lxd/db/containers.go:170]: Yes, I was going to perform file renames separately tho, if that's ok.

**Comment by stgraber** [comment]
Good we're doing this early in the cycle, backporting to 4.0 shouldn't be too bad.

**Comment by tomponline** [comment]
@freeekanayaka some of the unit tests are failing.

File: lxd/instances_post.go [hunk 2]
```diff
@@ -13,7 +13,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/dustinkirkland/golang-petname"
+	petname "github.com/dustinkirkland/golang-petname"
```
Comment (stgraber) [inline lxd/instances_post.go:16]: your editor having fun

File: lxd/instance/instance_utils.go [hunk 3]
```diff
@@ -12,7 +12,7 @@ import (
 
 	"github.com/pkg/errors"
 
-	"github.com/lxc/lxd/client"
+	lxd "github.com/lxc/lxd/client"
```
Comment (stgraber) [inline lxd/instance/instance_utils.go:15]: your editor having fun

**Comment by stgraber** [comment]
Looks good minus the two instances of your text editor adding aliases to imports when it's already their names :)

**Comment by freeekanayaka** [comment]
> Looks good minus the two instances of your text editor adding aliases to imports when it's already their names :)

I was using `goimports` when saving a .go file, which is a wrapper around `gofmt` that also fills in missing imports. Unfortunately there's no way to tell `goimports` to not rename packages, so I've disabled it for now and switched back just using plain `gofmt` for the time being.

**Pull request merged**